core/ordering/cosipbft/blockstore/genesis_test.go

```diff
@@ -88,17 +83,19 @@ func TestGenesisDiskStore_Load(t *testing.T) {
 	store.fac = fake.MessageFactory{}
 	err = store.Load()
 	require.EqualError(t, err, "unsupported message 'fake.Message'")
-}
 
-func TestGenesisDiskStore_Set(t *testing.T) {
-	dir, err := os.MkdirTemp(os.TempDir(), "dela-blockstore-")
+	err = db.Close()
 	require.NoError(t, err)
```

Comment:
This is inconsistent with the fix below, that looks better.
Do you want to `db.Close()` every time ? If so the `defer db.Close()` would make sense and avoid the DB remaining open if one of the other assertions fail.